src/concepts/sql_style.md

```diff
@@ -59,6 +63,45 @@ LIMIT
   10
 ```
 
+### Joins
+
+Always include the `JOIN` type rather than relying on the default join.
+
+**Good**
+```sql
+SELECT
+  submission_date,
+  e.key AS experiment_id,
+  e.value AS experiment_branch,
+  count(*) AS count
+FROM
+  telemetry.clients_daily
+CROSS JOIN
+  UNNEST(experiments) AS e(key, value)
```

Comment:
```suggestion
  UNNEST(experiments) AS experiment
```

I've seen several elegant unnest queries where the alias here is the non-plural version of the field name. I'd prefer `experiment` to `e` to encourage that style here, and perhaps even add a note encouraging that style, since UNNEST is going to be a common operation in the new BQ world.

I'm surprised to see the `e(key, value)` syntax. Is that actually renaming the fields in the struct?

Reply:
This is the way to do it on Presto and Athena. We should begin marking the bq SQL statements. @mreid-moz I do recommend using just the `experiment` and marking this as a BQ-query. The alternative is keeping as-is and marking as a Presto query, then we can update everything at once.

Reply:
Yep, this is Presto syntax, and you can rename fields when `UNNEST`ing a map to do things like this:

```sql
SELECT
  submission_date_s3,
  experiment.experiment_id,
  experiment.experiment_branch,
  count(*) AS count
FROM
  telemetry.clients_daily
CROSS JOIN
  UNNEST(experiments) AS experiment(experiment_id, experiment_branch)
WHERE
  submission_date_s3 > '20190703'
  AND sample_id = '10'
GROUP BY
  1, 2, 3
```

+1 for the `experiment` naming, and +1 for including the SQL "flavour", I will update.